Common/cpp/Tools/ReanimatedVersion.cpp

```diff
@@ -68,11 +76,16 @@ void checkJSVersion(
         "See `https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#mismatch-between-c-code-version-and-javascript-code-version` for more details.");
     return;
   }
+}
+#else // NDEBUG
```

Comment:
The comment is quite misleading, maybe let's get rid of it?
```suggestion
#else
```